From 55a7de27ea662e32d20717dec210053276f32a16 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Thu, 12 Nov 2020 19:52:02 +0100
Subject: [PATCH 1/1] efi_loader: missing \n in RNG log messages

When setting up the EFI_RNG_PROTOCOL warning messages may be issued.
The messages should end with a newline.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_loader/efi_rng.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/efi_loader/efi_rng.c b/lib/efi_loader/efi_rng.c
index a8a87007b6..8bdadad0a9 100644
--- a/lib/efi_loader/efi_rng.c
+++ b/lib/efi_loader/efi_rng.c
@@ -166,13 +166,13 @@ efi_status_t efi_rng_register(void)
 
 	ret = platform_get_rng_device(&dev);
 	if (ret != EFI_SUCCESS) {
-		log_warning("Missing RNG device for EFI_RNG_PROTOCOL");
+		log_warning("Missing RNG device for EFI_RNG_PROTOCOL\n");
 		return EFI_SUCCESS;
 	}
 	ret = efi_add_protocol(efi_root, &efi_guid_rng_protocol,
 			       (void *)&efi_rng_protocol);
 	if (ret != EFI_SUCCESS)
-		log_err("Cannot install EFI_RNG_PROTOCOL");
+		log_err("Cannot install EFI_RNG_PROTOCOL\n");
 
 	return ret;
 }
-- 
2.28.0

